node/service.json

```diff
@@ -4,5 +4,11 @@
   "ttl": 12,
   "timeout": 10,
   "minReplicas": 6,
-  "maxReplicas": 20
+  "maxReplicas": 20,
+  "routes": {
+    "viewerCountry": {
+      "path": "/_v/viewer-country",
```

Comment:
I would add the app name between `_v/` and the path like we do in other apps. This is a good practice to avid collision.